tests/system/Email/EmailTest.php

```diff
@@ -36,4 +38,42 @@ public function testEmailSendWithClearance($autoClear)
 			$this->assertEquals('foo@foo.com', $email->archive['recipients'][0]);
 		}
 	}
+
+	public function testSuccessTriggersEvent()
```

Comment:
need to have consistent grammar, trigger or triggers

Reply:
Done.